### PR TITLE
Code snippets

### PR DESCRIPTION
**Pull request by naterush**
# Description

Completes: https://www.notion.so/trymito/Code-Snippets-581c52cf00854476bac257a78e17e37c

Two questions:
1. Do we want some confirmation something was copied / written? Where should this go?
2. Should clicking on the code snippets do something?

# Testing

Test them, especially the integration.

# Documentation

Yep, we'll write notes for this feature on deployment.

### CODE REVIEW

File: mitosheet/src/components/icons/CodeSnippetIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+// Copyright (c) Mito
+
+import React from 'react';
+
+const CodeSnippetIcon = (): JSX.Element => {
+    return (
+        <svg width="20" height="20" viewBox="0 0 13 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (naterush) [inline mitosheet/src/components/icons/CodeSnippetIcon.tsx:8]: Lint

**Review by naterush — COMMENTED** [review]
Cleanup!

**Comment by aarondr77** [comment]
> Should clicking on the code snippets do something?

Updated so that clicking on the code snippet opens the dropdown menu + some highlighting on hover.

**Comment by aarondr77** [comment]
> Do we want some confirmation something was copied / written? Where should this go?

Added a confirmation message at the top of the taskpane that tells the user the next steps.

**Comment by aarondr77** [comment]
We should also move this code into the Enterprise folder and update the website

**Comment by naterush** [comment]
you rock

**Comment by naterush** [comment]
Looks great. LGTM. When we get more code snippets, we can change the location of that error messages to be in-line with the code snippet, so users always see it, but it's fine for now.

**Pull request merged**